tests/test_models.py

```diff
@@ -289,3 +295,52 @@ class TestRefreshTokenModel(TestCase):
     def test_str(self):
         refresh_token = RefreshToken(token="test_token")
         self.assertEqual("%s" % refresh_token, refresh_token.token)
+
+
+class TestClearExpired(TestCase):
+
+    def setUp(self):
+        self.user = UserModel.objects.create_user("test_user", "test@example.com", "123456")
+        """Insert two tokens on database."""
+        AccessToken.objects.create(
```

Comment:
Please use the project's existing style/format. e.g. with closing brackets...
```
AccessToken.objects.create(
    ...,
) 
```

Reply:
fixed